src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -309,23 +314,49 @@ const HealthCardDisplay = () => {
 
   return (
     <Grid container className={styles.root}>
-      {qrError ? (
-        <ErrorFallback error={qrError} />
-      ) : (
-        <ErrorBoundary FallbackComponent={ErrorFallback}>
-          <HealthCardDataProvider healthCardJws={jws}>
-            <Banners />
+      {(qrError)
+        ? (
+          <ErrorFallback error={qrError} />
+        ) : (
+          <ErrorBoundary FallbackComponent={ErrorFallback}>
+            <HealthCardDataProvider healthCardJws={jws[jws.length - 1]}>
+              <Banners />
+            </HealthCardDataProvider>
             <Grid item className={styles.flexCard}>
+              <Box style={{ textAlign: 'center' }}>
+                <Router>
+                  <Switch>
```

Comment:
The following warning is being thrown after scanning a QR code:

> Warning: React does not recognize the `computedMatch` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `computedmatch` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

It looks like this is because you have the Link component inside Switch (see: https://stackoverflow.com/questions/52916289/react-does-not-recognize-the-computedmatch-prop-on-a-dom-element)